### DISCUSSION TITLE
Update appears to download full package (instead of only differences)?

### DISCUSSION THREAD

**Discussion by jmbreuer** [legendary-gl/legendary#638] [Q&A] [answered]
I'm experiencing this with at least Bloons TD6 in Legendary, which had two (mandatory, ie won't launch without updating) updates within the last 5 days or so.

Running `legendary update Bloons` does perform the update correctly, but downloads between 1 and 1.5 GB of data each time - which corresponds to the download size of a full fresh install, at first glance.

Is this a peculiar artifact, a known limitation, or something else?

**Comment by derrod** [comment]
Seems they renamed some files for some reason, which means the chunks won't be reused but redownloaded.

**Reply by nutterthanos** [reply, marked as answer]
yeah being the game is in unity and being it has .bundle files with different file names every update
in StreamingAssets/aa~/StandaloneWindows64/

**Reply by jmbreuer** [reply]
Aah thanks!

**Reply by nutterthanos** [reply]
usually some chunks from older .bundle files should pass through but a lot of the time they don't
